include/sys/util.h

```diff
@@ -204,6 +204,21 @@ extern "C" {
 #define CLAMP(val, low, high) (((val) <= (low)) ? (low) : MIN(val, high))
 #endif
 
+/**
+ * @def IN_RANGE
```

Comment:
```suggestion
```
this is redundant, should be deleted

Reply:
This was just added to be consistent with MIN, MAX and CLAMP, but was removed.

Reply:
FYI https://github.com/zephyrproject-rtos/zephyr/pull/43582